src/transformers/models/dpt/modeling_flax_dpt.py

```diff
@@ -0,0 +1,1236 @@
+        fused_hidden_states.append(fused_hidden_state)
+        # looping from the last layer to the second
+        for i, hidden_state in enumerate(hidden_states[1:]):
+            fused_hidden_state = self.layers(fused_hidden_state, residual=hidden_state, i=i + 1)
+            fused_hidden_states.append(fused_hidden_state)
+        return fused_hidden_states
+
+
+class FlaxDPTPreActResidualLayer(nn.Module):
+    config: DPTConfig
+    dtype: jnp.dtype = jnp.float32
+
+    def setup(self):
+
+        self.use_batch_norm = self.config.use_batch_norm_in_fusion_residual
+        self.activation1 = ACT2FN["relu"]
+        self.convolution1 = nn.Conv(
+            self.config.fusion_hidden_size,
+            kernel_size=(3, 3),
+            strides=(1, 1),
+            padding="SAME",
+            use_bias=not self.use_batch_norm,
+        )
+
+        self.activation2 = ACT2FN["relu"]
+        self.convolution2 = nn.Conv(
+            self.config.fusion_hidden_size,
+            kernel_size=(3, 3),
+            strides=(1, 1),
+            padding="SAME",
+            use_bias=not self.use_batch_norm,
+        )
+
+        if self.use_batch_norm:
+            self.batch_norm1 = nn.BatchNorm(use_running_average=True)
+            self.batch_norm2 = nn.BatchNorm(use_running_average=True)
+
+    def __call__(self, hidden_state):
+        residual = hidden_state
+        hidden_state = self.activation1(hidden_state)
+
+        hidden_state = self.convolution1(hidden_state)
+
+        if self.use_batch_norm:
+            hidden_state = self.batch_norm1(hidden_state)
+
+        hidden_state = self.activation2(hidden_state)
+        hidden_state = self.convolution2(hidden_state)
+
+        if self.use_batch_norm:
+            hidden_state = self.batch_norm2(hidden_state)
+
+        return hidden_state + residual
+
+
+class FlaxDPTFeatureFusionLayerCollection(nn.Module):
+    config: DPTConfig
+    dtype: jnp.dtype = jnp.float32
+
+    def setup(self):
+        self.layers = [
+            FlaxDPTFeatureFusionLayer(self.config, name=str(i)) for i in range(len(self.config.neck_hidden_sizes))
+        ]
+
+    def __call__(self, hidden_states, residual=None, i=0):
+        return self.layers[i](hidden_states, residual)
+
+
+class FlaxDPTFeatureFusionLayer(nn.Module):
+    config: DPTConfig
+    dtype: jnp.dtype = jnp.float32
+    align_corners: bool = True
+
+    def setup(self):
+        self.projection = nn.Conv(self.config.fusion_hidden_size, kernel_size=(1, 1))  # , bias=True)
+
+        self.residual_layer1 = FlaxDPTPreActResidualLayer(self.config)
+        self.residual_layer2 = FlaxDPTPreActResidualLayer(self.config)
+        self.upsample = FlaxDPTUpsample()
+
+    def __call__(self, hidden_state, residual=None):
+        if residual is not None:
+            if hidden_state.shape != residual.shape:
+                size = hidden_state.shape
+                residual = self.upsample(residual, size)
```

Comment:
Set  `align_corners=True`
```suggestion
                residual = self.upsample(residual, align_corners=True)
```

Reply:
Fixed in 791ea245ed3470463d3a9e6fee2c49d384a4a71a